datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/settings/applications/UpdateApplicationsSettingsResolver.java

```diff
@@ -0,0 +1,75 @@
+package com.linkedin.datahub.graphql.resolvers.settings.applications;
+
+import static com.linkedin.datahub.graphql.resolvers.ResolverUtils.*;
+
+import com.linkedin.datahub.graphql.QueryContext;
+import com.linkedin.datahub.graphql.authorization.AuthorizationUtils;
+import com.linkedin.datahub.graphql.concurrency.GraphQLConcurrencyUtils;
+import com.linkedin.datahub.graphql.exception.AuthorizationException;
+import com.linkedin.datahub.graphql.generated.UpdateApplicationsSettingsInput;
+import com.linkedin.metadata.service.SettingsService;
+import com.linkedin.settings.global.GlobalSettingsInfo;
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import java.util.Objects;
+import java.util.concurrent.CompletableFuture;
+import javax.annotation.Nonnull;
+
+/** Resolver responsible for updating the actions settings. */
+public class UpdateApplicationsSettingsResolver implements DataFetcher<CompletableFuture<Boolean>> {
+
+  private final SettingsService _settingsService;
+
+  public UpdateApplicationsSettingsResolver(@Nonnull final SettingsService settingsService) {
```

Comment:
i think we can add some resolver tests for this bad boy

Reply:
addressed!